RADAR/donations.yml

```diff
@@ -0,0 +1,34 @@
+- ticker: RADAR
+         
+  recipient: 
+    name: Coin Center
+    address: 700 K Street NW, Suite 300, Washington D.C., 20001 USA
+    tel: 202-804-6013
+    contact: info@coincenter.org
+
+  ada: 50
+  usd: 137.50
+  timestamp:	1629935090 
+  # ^ This is a unix timestamp of the moment you donated.
+  #   You can go to https://www.unixtimestamp.com/ to calculate your donation.
+
+  files: RADAR/RADAR Coin Center Sept 2021.txt
+ 
+ 
+ - ticker: RADAR
+         
+  recipient: 
+    name: Coin Center
+    address: 700 K Street NW, Suite 300, Washington D.C., 20001 USA
+    tel: 202-804-6013
+    contact: info@coincenter.org
+
+  ada: 151
+  usd: 337.13
+  timestamp:	 
```

Comment:
Please add a timestamp

Reply:
OK. I think I've cleaned up my donations.yml file.
Should finally be good for Sept and Oct.  
Please let me know